<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardMachineCalibrateDialog</class>
 <widget class="QDialog" name="StandardMachineCalibrateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>811</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="5">
     <widget class="QLineEdit" name="rightAdditionStepLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="CapTypeComboBox">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>35</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>T-Cup</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cube-Cup</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leftLightLowlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="leftLabel2">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="downLightHighlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>25000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="leftLabel1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>左绿灯亮度范围：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="downLabel2">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="downLabel1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>下绿灯亮度范围：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="downLightLowlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="leftLightHighlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>25000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="upperLightHighlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>25000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="downAdditionStepLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="upperLightLowlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="leftAdditionStepLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="upperLabel1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>上绿灯亮度范围：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="rightLightLowlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="upperLabel2">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="rightLabel1_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>杯型：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="downLabel3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>步进亮度：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="upperAdditionStepLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="upperLabel3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>步进亮度：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="rightLabel2">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="leftLabel3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>步进亮度：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="rightLabel1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>右绿灯亮度范围：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="rightLightHighlineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>25000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="rightLabel3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>步进亮度：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="testCountLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>单个亮度测试次数：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="testCountLineEdit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
